src/components/TaxPayer/SpouseAndDependent.tsx

```diff
@@ -75,44 +78,67 @@ export const AddDependentForm = (): ReactElement => {
 
 export const SpouseInfo = (): ReactElement => {
   const { register, control, errors, handleSubmit, getValues } = useForm<UserPersonForm>()
-  const [editSpouse, updateEditSpouse] = useState(false)
+  const [editSpouse, setEditSpouse] = useState<boolean>(false)
+  const [createSpouse, setCreateSpouse] = useState<boolean>(false)
```

Comment:
It might be nice to abstract these (and other?) hooks into custom hooks

Reply:
Yeah that would b nice to have, not sure of a way to abstract further though